src/rust/ide/view/graph-editor/src/component/visualization/instance.rs

```diff
@@ -32,42 +34,46 @@ pub struct Frp {
     #[shrinkwrap(main_field)]
     pub inputs                 : FrpInputs,
 
-    pub on_preprocessor_change : frp::Stream<EnsoCode>,
+    pub on_preprocessor_change : frp::Stream<enso::Code>,
     pub on_data_receive_error  : frp::Stream<Option<DataError>>,
     pub is_active              : frp::Stream<bool>,
 
+    /// This event should be emitted when the received data are incorrect, or cause an internal
+    /// error.
     pub data_receive_error    : frp::Source<Option<DataError>>,
-    pub preprocessor_change: frp::Source<EnsoCode>,
-    pub activate              : frp::Source,
-    pub deactivate            : frp::Source,
+    /// This event should be emitted  should be emitted to set a new code of the preprocessor. The
+    /// preprocessor is a function called on the Engine side before sending data to IDE, allowing us
+    /// do some compression or filtering for the best performance. See also _Lazy Visualization_
+    /// section in `docs/product/visualizations.md`
```

Comment:
Is there special syntax to create a doc link here?

Reply:
I don't know, but I put here a simple link to dev.enso.org